orchestration/docker.go

```diff
@@ -0,0 +1,216 @@
+package orchestration
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/joho/godotenv"
+)
+
+func runNodeDocker(nodeType string, network string, imageTag string, autoUpgrade bool) {
+	fmt.Printf("standing up %s on network %s\n", nodeType, network)
+	volumeFlag := ""
+	//volumeFlag = fmt.Sprintf("-v %s:/root/audius-docker-compose/%s/override.env", confFilePath, nodeType)
```

Comment:
This does not work currently with the switch to toml configs. Purposefully waiting for the config changes from @alecsavvy before integrating a solution.